config/v1.7/calico.yaml

```diff
@@ -32,7 +32,12 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: quay.io/calico/node:v3.15.1
```

Comment:
Why do we have calico config changes?

Reply:
We should have calico copied from master to v1.7. Seems we didn't do that in previous release?

Reply:
Master/calico has some changes which needs to be merged to release-1.7. Changes will be merge to release-1.7 in the next release.